MS-DOS-6.22 libi86 item  5 C compiler toolchain test times out (regression)

**Describe the bug**
`test/test_dosemu.py MSDOS622TestCase.test_libi86_item_005` times out (with or without KVM), as reported by CI in https://github.com/dosemu2/dosemu2/pull/2775

**To Reproduce**
```
$ test/test_dosemu.py MSDOS622TestCase.test_libi86_item_005
 
Usable KVM not found, falling back to emulation: 
Test MS-DOS-6.22 libi86 item   5 C compiler toolchain                            ... FAIL
 
======================================================================
FAIL: Test MS-DOS-6.22 libi86 item   5 C compiler toolchain                           
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/oldeman/dosemu2/test/func_libi86_testsuite.py", line 36, in do_test_libi86
    libi86_test_item(self, test[0])
    ~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
  File "/home/oldeman/dosemu2/test/func_libi86_testsuite.py", line 76, in libi86_test_item
    raise self.failureException("Test timeout") from None
AssertionError: Test timeout
Further info in file 'test_dosemu.MSDOS622TestCase.test_libi86_item_005.log'
Further info in file 'test_dosemu.MSDOS622TestCase.test_libi86_item_005.xpt'
 
----------------------------------------------------------------------
Ran 1 test in 60.012s
 
FAILED (failures=1)
```
same with KVM enabled, except it states `Usable KVM found` instead


**A regression?**
Yes,  260b283f

--- a/src/base/core/int.c
+++ b/src/base/core/int.c
@@ -4100,29 +4100,3 @@ static int msdos_remap_extended_open(void)
     CARRY;
   return ret;
 }
-
-far_t get_int_vector(int vec)
-{
-    far_t addr;
-
-    if (test_bit(vec, &vm86s.int_revectored)) {
-	addr.segment = INT_RVC_SEG;
-	switch (vec) {
-	case 0x21:
-	    int21_rvc_setup();
-	    addr.offset = INT_RVC_21_OFF;
-	    return addr;
-	case 0x2f:
-	    int2f_rvc_setup();
-	    addr.offset = INT_RVC_2f_OFF;
-	    return addr;
-	case 0x33:
-	    int33_rvc_setup();
-	    addr.offset = INT_RVC_33_OFF;
-	    return addr;
-	}
-    }
-    addr.segment = ISEG(vec);
-    addr.offset = IOFF(vec);
-    return addr;
-}

--- a/src/dosext/dpmi/dpmi.c
+++ b/src/dosext/dpmi/dpmi.c
@@ -2016,7 +2016,11 @@ int DPMI_allocate_specific_ldt_descriptor(unsigned short selector)
 
 far_t DPMI_get_real_mode_interrupt_vector(int vec)
 {
-    return get_int_vector(vec);
+    far_t addr;
+
+    addr.segment = ISEG(vec);
+    addr.offset = IOFF(vec);
+    return addr;
 }
 
 int DPMIAllocateShared(struct SHM_desc *shm)

--- a/src/include/int.h
+++ b/src/include/int.h
@@ -35,7 +35,6 @@ void int_try_disable_revect(void);
 enum { I_NOT_HANDLED, I_HANDLED, I_SECOND_REVECT };
 
 extern int can_revector(int i);
-far_t get_int_vector(int vec);
 
 void update_xtitle(void);
 

